inst/template/dust.R.template

```diff
@@ -452,8 +452,12 @@
     ##' parameter sets.
     ##'
     ##' @param data A list of data to set.
-    set_data = function(data) {
-      private$methods_$set_data(private$ptr_, data)
+    ##'
+    ##' @param shared Logical, indicating if the data should be shared
```

Comment:
`shared_data` here too?

Reply:
I've updated the C++ facing ones, but not this one as the user here should not be thinking about shared memory